docs/product-user-guide/src/main/asciidoc/interacting-with-processes-starting-stopping-tasks-proc.adoc

```diff
@@ -12,16 +12,16 @@ Ensure that you are logged in and have appropriate permissions for starting and
 == Starting a task
 
 .Procedure
-. Go to *Menu* -> *Manage* -> *Tasks*.
-. On the *Manage Tasks* page, click on the task to open it.
-. On the task page, click *Start*. Once you start a task, its status changes to *InProgress*.
+. Go to *Menu* -> *Track* -> *Task Inbox*.
+. On the *Task Inbox* page, click on the task to open it.
+. On the *Work* tab of the task page, click *Start*. Once you start a task, its status changes to *InProgress*.
 
- You can view the status of tasks on the *Manage Tasks* page.
+ You can view the status of tasks on the *Task Inbox* as well as on the *Manage Tasks* page.
```

Comment:
By default, only process admins and admins can access the _Manage_ section. In the context of the knowledge worker, its probably not relevant but if we still want to describe, the access restriction should be noted as well.